plugin.yml

```diff
@@ -0,0 +1,23 @@
+name: Smooth Secrets
+description: Plugin to manage your secrets
+author: https://github.com/hasura
+requirements: []
+configuration:
+  properties:
+    secrets:
+      type: object
```

Comment:
we should probably denote this to be an array of objects with the following properties